mesa_geo/geospace.py

```diff
@@ -120,12 +120,12 @@ def agents_at(self, pos):
 
     def distance(self, agent_a, agent_b):
         """Return distance of two agents."""
-        return agent_a.shape.distance(agent_b.shape)
+        return agent_a.geometry.distance(agent_b.geometry)
 
     def _create_neighborhood(self):
         """Create a neighborhood graph of all agents."""
         agents = self.agents
-        shapes = [agent.shape for agent in agents]
+        shapes = [agent.geometry for agent in agents]
```

Comment:
Seems to make more sense to call it `geometries`?

Reply:
Totally! I should've changed it.